sdk/program/src/stake/mod.rs

```diff
@@ -6,7 +6,8 @@ pub mod program {
     crate::declare_id!("Stake11111111111111111111111111111111111111");
 }
 
-/// The minimum stake amount that can be delegated, in lamports.
-/// NOTE: This is also used to calculate the minimum balance of a stake account, which is the
-/// rent exempt reserve _plus_ the minimum stake delegation.
+#[deprecated(
+    since = "1.10.6",
```

Comment:
What's the process for deciding which version a feature should land in? I was originally thinking that I would backport this PR to v1.10, but is that right? @jstarry wdyt?

Reply:
i always use the current version of the oldest branch i intend to backport to

Reply:
> i always use the current version of the oldest branch i intend to backport to

Nice, makes sense!

> And since this PR could be split up into three separate pieces, maybe the different pieces have different backport answers

This particular deprecation is specific to this PR so should be fine to follow trent's suggestion